<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Separation</class>
 <widget class="QWidget" name="Main_Separation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}
QLabel
{
	border:1px solid lightgray;
    color:white;
font:16px
}

QPushButton
{
	font:20px;
    color: rgb(240,240,240);
	background-color: rgb(0, 0, 0);
	border-top: 2px solid white;	
	border-left: 2px solid white;
   border-right: 1px solid white;
   border-bottom: 1px solid white;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>361</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_8">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_C">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_0">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_9">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_T_1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>自动过分相测试激活</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_T_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>自动过分相测试取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_number">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Send">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
